torch/csrc/jit/autodiff.h

```diff
@@ -45,7 +45,7 @@ struct Gradient {
   // Describes how to construct outputs of f from what its graph will return.
   // This is necessary because some trailing outputs are intermediates produced
   // only to be saved for df (and should be ignored).
-  size_t f_real_outputs;
+  size_t f_real_outputs = 0;
```

Comment:
Uhhhh what... That doesn't make sense semantically and `0` is no better than a random value that we read from memory there. Such changes only make the code confusing by making it appear as this was a deliberately set value, while it doesn't make any sense.